FlowCrypt/Functionality/Services/GoogleUserService.swift

```diff
@@ -229,6 +243,55 @@ extension GoogleUserService {
     }
 }
 
+// MARK: - Tokens
+extension GoogleUserService {
+    func getIdToken() async throws -> String {
```

Comment:
Let's call it `getCachedOrRefreshedIdToken` to make it clear an external call may happen ?